97.md

```diff
@@ -0,0 +1,251 @@
+NIP-97
+=======
+
+Attachments
+-----------
+
+`final` `optional` `author:ondra-novak`
+
+This NIP solves the problem of sharing and distributing binary content over NOSTR networks such as media - images, short videos music, or in general any files up to a certain size
+
+Key Takeaways
+-------------
+
+* Defines how binary content (images, media, files) is managed by NOSTR network
+* Allows sending private binary content in direct messages (encrypted)
+* Defines new commands at the protocol level ("ATTACH","FETCH")
+* Defines a new tag "attachment"
+* Uses the binary messages in the websocket connection
+
+
+New tag "attachment"
+-------------
+The proposal introduces a new tag  "attachment" with the following format
+
+```
+["attachment","hash_hex","size_in_bytes","mime_type","features1", "feature2", ....]
```

Comment:
You could use a FileMetadata Event instead of this tag. https://github.com/nostr-protocol/nips/blob/master/94.md  You can just use the already common `x` tag to be the hash of the attachment. There is no need to redefine all possible metadata tags here again. 

Additionally, in that way, we don't run the risk of breaking relays out there that don't support infinite tag columns.

Reply:
This is replied in FAQ section

Reply:
I think all relays that use a relational database have a limit on the number of columns. We just don't know what those are :(